library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -954,6 +954,7 @@ def Write(devregex)
 
       # Finish him
       SCR.Write(path(".network"), nil)
+      CleanCacheRead()
```

Comment:
please add comment why it is needed.

Reply:
Sure, I was still working on it, that is why it is still in draft :)